### PR TITLE
Add PR template and auto-close PR on subtree split repositories

### PR DESCRIPTION
**Pull request by GromNaN**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | Fix #57267
| License       | MIT

Extract the Pull Request Template from #54653.

~The GitHub action did not work as said by @nicolas-grekas https://github.com/symfony/symfony/pull/57267#issuecomment-2142429561.~ Replaced by https://github.com/superbrothers/close-pull-request.

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
From https://docs.github.com/en/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/approving-workflow-runs-from-public-forks

>Workflows triggered by pull_request_target events are run in the context of the base branch. Since the base branch is considered trusted, workflows triggered by these events will always run, regardless of approval settings. For more information about the pull_request_target event, see "[Events that trigger workflows](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target)."

I'm not sure why this didn't work. We might try on a dummy repo to confirm which approachs works?

**Comment by dunglas** [comment]
For API Platform, we use https://github.com/superbrothers/close-pull-request.

Laravel uses the same action. It works well.

**Comment by GromNaN** [comment]
Adding the close-pull-request action, as it seems to work even for new contributor. On the other hand, @nicolas-grekas your script was expecting the GitHub credentials, that could explain why it's blocked for new contributors.

File: .github/sync-packages.php [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<?php
+
+if ('cli' !== PHP_SAPI) {
+    echo "This script can only be run from the command line.\n";
+    exit(1);
```
Comment (nicolas-grekas) [inline .github/sync-packages.php:5]: script should be removed I guess?

Comment (GromNaN) [reply .github/sync-packages.php:5]: It should be run on newest branches for new components. And can be used to update the files quickly.

**Review by nicolas-grekas — APPROVED** [review]

**Review by javiereguiluz — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @GromNaN.

**Comment by nicolas-grekas** [comment]
merged up to 7.2

**Pull request merged**